features/step_definitions/checks_for_rails_best_practices_steps.rb

```diff
@@ -1,5 +1,5 @@
 def rails_best_practices_cmd
-  'rails_best_practices --silent --spec --test --features .'
+  'rails_best_practices --without-color --silent --spec --test --features .'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.